api/app.py

```diff
@@ -86,7 +87,21 @@ def create_app(test_config=None) -> Flask:
 
     app.secret_key = app.config['SECRET_KEY']
 
-    logging.basicConfig(level=app.config.get('LOG_LEVEL', 'INFO'))
+    log_file = app.config.get('LOG_FILE')
```

Comment:
If LOG_FILE is empty, logs will be output only to stdout in the new commit. Please help review it.

Reply:
I believe that the code app.config.get('LOG_LEVEL', 'INFO') can be misleading to developers, because the 'LOG_LEVEL' key is defined in config.py (even if it is None or an empty string), and the default value afterwards will only be returned when you haven't defined this key (which can be misleading).
So I changed this section to app.config.get('LOG_LEVEL'), and the default value should be modified in the DEFAULTS variable in config.py to take effect.

Reply:
Customizing the configuration is fine, but we recommend modifying the configuration values in `.env` rather than changing the contents in config.py `DEFAULTS`.

Reply:
The new version has set the LOG_FILE key in the DEFAULTS variable of the config.py file to an empty string. Could you please take a look when you have time to see if anything else needs to be changed?